lr-cli/cortx_setup/commands/storage/config.py

```diff
@@ -107,6 +107,24 @@ class StorageEnclosureConfig(Command):
             'default': None,
             'optional': True,
             'help': 'Port of the controller to connect to'
+        },
+        'cvg': {
+            'type': int,
+            'default': None,
+            'optional': True,
+            'help': 'cvg number'
```

Comment:
Change help message:
Number of cylinder volume groups

Reply:
As discussed in the meeting, it's not no of cylinder volume groups it's the index, so added help as:
"Cyllinder Volume Group number." Hope that's fine.